pkg/oc/bootstrap/clusterup/componentinstall/privileged.go

```diff
@@ -0,0 +1,36 @@
+package componentinstall
+
+import (
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apiserver/pkg/authentication/serviceaccount"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/util/retry"
+
+	"github.com/openshift/origin/pkg/oc/errors"
+	securityclient "github.com/openshift/origin/pkg/security/generated/internalclientset/typed/security/internalversion"
+)
+
+func AddPrivilegedUser(clientConfig *rest.Config, namespace, name string) error {
```

Comment:
@openshift/sig-security how is the "SCC allowed by rolebinding" coming along?

Reply:
@deads2k It was brought up on today's planning. @enj is going to create a Trello card for that and I will implement it.

Reply:
> @openshift/sig-security how is the "SCC allowed by rolebinding" coming along?
>

> @deads2k It was brought up on today's planning. @enj is going to create a Trello card for that and I will implement it.
> 
> Write

@smarterclayton great news for us!